src/css/styles.css

```diff
@@ -79,57 +93,227 @@ body {
     background-color: rgba(139, 227, 255, 0.46);
 }
 
+.pager__previous {
+    margin-right: 5px;
+    width: 50%;
+}
+
+.pager__previous .fa {
+    margin-right: 0;
+}
+
+.pager__next {
+    width: 48%;
+}
+
+/* Change View */
+.change-view-container {
+    display: flex;
+    flex-direction: column;
+}
 
+.change-view-actions {
+    border-bottom: 1px solid #eee;
+    border-top: 1px solid #eee;
+    margin: 1em 0 0.5em 0;
+    padding: 0.7em 0.2em;
+}
+
+.change-view-actions > div {
+    padding: 0;
+}
+
+.change-view-actions .btn-link {
+    border: none;
+    padding: 0;
+    font-size: inherit;
+    display: inline;
+    line-height: inherit;
+    vertical-align: baseline;
+}
+
+/* Version Selector */
+.version-selector {
+    display: flex;
+    justify-content: space-between;
+    margin: 0.5em 0;
+}
+
+.version-selector__item {
+    font-weight: normal;
+}
+
+.version-selector__item > span {
+    font-weight: bold;
+    margin-right: 0.5em;
+}
+/* END - Version Selector */
+
+/* ===== Annotation Form ===== */
+.lnk-action {
+    margin-right: 17px;
+}
 
 .annotation-form {
-    margin-bottom: 10px;
-    margin-top: 10px;
+    margin: 0.4em 0;
 }
 
-.annotation-form input[type="checkbox"] {
-    cursor: pointer;
-    margin-right: 7px;
+.annotation-form__signifiers {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 0.5em;
+    width: 100%;
 }
 
-.annotation-form label {
-    display: block;
-    font-weight: 200;
-    font-size: 1.4rem;
-    cursor: pointer;
+.annotation-form textarea {
+    border: 1px solid #c3c3c3;
+    border-radius: 3px;
+    color: #555;
+    height: 3em;
+    padding: 0.5em;
+    width: 100%;
+}
+
+.signifier-container h5 {
+    font-size: 12px;
+    margin: 0 0 7px 0;
+}
+
+.signifier-list, .signifier-list-item {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.signifier-list {
+    display: flex;
+    flex-direction: column;
+}
+
+/* Remove margin from last signifier button  */
+.signifier-list > li:last-child {
+    margin: 0;
+}
+
+.signifier-list-item {
+    display: flex;
+    margin-right: 5px;
+}
+
+.info-text {
+    font-size: 12px;
+    margin: 5px 0 0 5px;
+}
+
+/* Hide tooltips when not collapsed */
+.__react_component_tooltip {
+    display: none!important;
+}
+
+.annotation-form--collapsed .__react_component_tooltip {
+    display: block!important;
+}
```

Comment:
It seems like we’d be better off to add a `--expanded` class and use that to create just one rule instead of two. Alternatively, you could also just not render the`<Tooltip>` element over on the JS side if the form is expanded.